packages/mobile/src/localCurrency/saga.ts

```diff
@@ -59,7 +58,7 @@ export function* watchFetchCurrentRate() {
 
 export function* watchSelectPreferredCurrency() {
   while (true) {
-    yield take([Actions.SELECT_PREFERRED_CURRENCY, REHYDRATE, AccountActions.SET_PHONE_NUMBER])
+    yield take([Actions.SELECT_PREFERRED_CURRENCY, AccountActions.SET_PHONE_NUMBER])
```

Comment:
why remove this here? this will ensure we fetch the local currency on app startup

Reply:
It removes Rehydrate from being one of the required actions before it does it. We now take a rehydrate before running *any* sagas.

Reply:
Fixed